src/js/tests/functions.test.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable no-useless-concat */
+/* eslint-disable global-require */
```

Comment:
- [ ] Kindly do not disable linters, remove these lines and fix your code in order to pass the linter checks in the Github PR. The linters help to ensure that your code does not include any structural issues which can make your code harder to maintain.